bedwars-plugin/src/main/java/com/tomkeuper/bedwars/arena/Misc.java

```diff
@@ -33,8 +33,6 @@
 import com.tomkeuper.bedwars.configuration.Sounds;
 import com.tomkeuper.bedwars.stats.PlayerStats;
 import com.tomkeuper.bedwars.support.papi.SupportPAPI;
-import com.google.common.io.ByteArrayDataOutput;
-import com.google.common.io.ByteStreams;
```

Comment:
Aahh you removed it here and then re-added it at the top

Reply:
> Aahh you removed it here and then re-added it at the top

Looks like I used the Optimize Imports feature